packages/plugin-fontawesome/src/FontAwesomeDomObjectRenderer.ts

```diff
@@ -20,11 +20,28 @@ export class FontAwesomeDomObjectRenderer extends NodeRenderer<DomObject> {
         node: FontAwesomeNode,
         worker: RenderingEngineWorker<DomObject>,
     ): Promise<DomObject> {
+        const select = (ev: Event) => {
+            ev.preventDefault();
+            const selectFontAwesome = () => {
+                this.engine.editor.selection.select(node, node);
+            };
+            // The normaliser wait a tick before changing the selection when the user click
+            // in order to override the normalizer default behavior we also need to wait a tick
+            setTimeout(() => {
+                this.engine.editor.execCommand(selectFontAwesome);
+            });
```

Comment:
This can be wrong in case of concurrency issue.
Imagine that a user do a batch of event like:
[select, keydown, ...] in one tick, then the selection will be wrong as it happens after many event occurred.

Reply:
Yeah, I'm not a fan either, But how can I do this differently ?

Reply:
Also, it is wrong as it does not prevent the `Normalizer`, it rather tries to **correct** it. Which is wrong.
We want to correctly prevent the `Normalizer` for the `mousedown` to not trigger a `setSelection`.

We need a mechanism in the `Normalizer` for that.
Like when there is a `mousedown`, we need to wait if there was no (click/double click/triple click) that has followed.

Reply:
As I changed related code in my PR "concurency", if we want that mechanism, we need to wait the PR to be merged.

Reply:
We can wait if your changes make this better

Reply:
I think this might be a case where it is worth it to wait for the concurrency PR to be merged.

Reply:
:+1: